include/common/pca_9685.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+#ifdef __linux__
+
+// Standard C includes
+#include <cstdint>
+
+// BoB robotics third party includes
+#include "third_party/units.h"
+
+// BoB robotics includes
+#include "i2c_interface.h"
+
+//----------------------------------------------------------------------------
+// BoBRobotics::PCA9685
+//----------------------------------------------------------------------------
+//! Userspace drive for PCA9685 pulse-width modulation (PWM) chip
+//! Commonly used for controlling servos, LEDs and motors
+namespace BoBRobotics {
+class PCA9685
+{
+public:
+    PCA9685(int slaveAddress, const char *path = I2C_DEVICE_DEFAULT, 
+            units::frequency::hertz_t referenceClockSpeed = units::frequency::megahertz_t(25.0));
+    
+    //----------------------------------------------------------------------------
+    // Enumerations
+    //----------------------------------------------------------------------------
+    enum class Channel : uint8_t
+    {
+        CHANNEL_0 = 0,
+        CHANNEL_1 = 1,
+        CHANNEL_2 = 2,
+        CHANNEL_3 = 3,
+        CHANNEL_4 = 4,
+        CHANNEL_5 = 5,
+        CHANNEL_6 = 6,
+        CHANNEL_7 = 7,
+        CHANNEL_8 = 8,
+        CHANNEL_9 = 9,
+        CHANNEL_10 = 10,
+        CHANNEL_11 = 11,
+        CHANNEL_12 = 12,
+        CHANNEL_13 = 13,
+        CHANNEL_14 = 14,
+        CHANNEL_15 = 15,
+    };
```

Comment:
I feel like having this as an ``enum`` is a bit unnecessary, but it's harmless I guess

Reply:
It just means you don't need to assert somewhere that channel >= 1 and channel <= 15 :man_shrugging: